pinot-core/src/main/java/org/apache/pinot/core/realtime/impl/RealtimeSegmentConfig.java

```diff
@@ -47,15 +48,16 @@
   private final boolean _nullHandlingEnabled;
   private final String _consumerDir;
 
-  private RealtimeSegmentConfig(String segmentName, String streamName, Schema schema, int capacity,
-      int avgNumMultiValues, Set<String> noDictionaryColumns, Set<String> varLengthDictionaryColumns,
+  private RealtimeSegmentConfig(String segmentName, String streamName, Schema schema, String timeColumnName,
```

Comment:
Can you add a TODO here to clean this up with table config?

Reply:
Done